misk-crypto/src/main/kotlin/misk/crypto/Cipher.kt

```diff
@@ -0,0 +1,69 @@
+package misk.crypto
+
+import com.google.crypto.tink.Aead
+import com.google.crypto.tink.JsonKeysetWriter
+import com.google.crypto.tink.KeysetHandle
+import com.google.crypto.tink.aead.AeadFactory
+import okio.ByteString
+import okio.ByteString.Companion.toByteString
+import java.io.ByteArrayOutputStream
+import java.util.Base64
+
+/**
+ * An even simpler interface than Tink's [Aead].
+ * This also makes it easier to mock Ciphers for testing purposes.
+ */
+interface Cipher {
+  /**
+   * Encrypt the given data.
+   * If anything goes wrong, throws a [java.security.GeneralSecurityException]
```

Comment:
Is it possible for anything to go wrong?

Reply:
This seems to me like one of those things where theoretically the VM might not have AEAD but in practice it never happens, and it’s a waste to write code to recover from that. 

(So don’t document it)